modules/releng.gradle

```diff
@@ -343,6 +344,80 @@ prepareMajorIncrement {
 	}
 }
 
+tagAppReleases {
+	description = "Gathers the Git IDs from all app changelog files and tags them as 'marketplace-appName-appVersion'."
+
+	doLast {
+		FileTree appChangelogFileTree = fileTree(dir: projectDir, includes: [".releng/apps/**/app.changelog"])
+
+		appChangelogFileTree.each {
+			appChangelog ->
+
+			String appName = appChangelog.getParentFile().getName()
+
+			String appChangelogTextFiltered = appChangelog.filterLine {
+				it =~ /app\.git\.id/
+			}
+
+			List appGitIDLines = appChangelogTextFiltered.readLines()
+
+			for (appGitIDLine in appGitIDLines) {
+				String appGitID = appGitIDLine.replaceFirst(/app\.git\.id-.+=/, "")
+
+				String appVersion = appGitIDLine.replaceFirst(/app\.git\.id-(.+)=.+/, /$1/)
+
+				String appGitTagName = "marketplace-" + appName + "-" + appVersion
+
+				String gitRevListCmd = "git rev-list -1 " + appGitTagName
+
+				Process process = gitRevListCmd.execute()
```

Comment:
Since we're already using [grgit](https://github.com/ajoberstar/grgit), can't we avoid calling Git directly? Do you think the `_getRevCommit` method does what you need?